cwd/editor.js

```diff
@@ -62,6 +62,9 @@ views.forEach(view => {$.getJSON(view, function(data) {
       const viewNum = this.id.substring(8,9);
       const msgNum = this.id.substring(10,11);
 
+      const prob = [];
+      prob.push($('#prob-' + (this.id.substring(8,)) + "-" + 1 ).val());
```

Comment:
Does this handle views' message probabilities being a single integer fine?  If probability is an array, I believe the program starts looking for attached data.  I'll test this soon.

Reply:
This has been resolved, I believe.